webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                        ],
+                        value="exc",
+                        labelStyle={"display": "inline-block"},
+                        style={"fontSize": ".80em"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("parameter-list"),
+                        options=[{"label": ens, "value": ens} for ens in self.parameters],
+                        clearable=True,
+                        multi=True,
+                        placeholder="",
+                        value=[],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div("Filters:", style={"font-weight": "bold"}),
+            html.Div(children=self.filter_layout),
+            html.Div(
+                [
+                    html.Div(
+                        "Model settings:", style={"font-weight": "bold", "marginTop": "20px"},
+                    ),
+                    html.Div(
+                        "Interaction", style={"display": "inline-block", "margin-right": "10px"},
+                    ),
+                    dcc.Slider(
+                        id=self.uuid("interaction"),
+                        min=0,
+                        max=2,
+                        step=None,
+                        marks={0: "Off", 1: "2 levels", 2: "3 levels"},
+                        value=0,
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div(
+                        "Max number of parameters",
+                        style={"display": "inline-block", "margin-right": "10px"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("max-params"),
+                        options=[
+                            {"label": val, "value": val}
+                            for val in range(1, min(10, len(self.parameterdf.columns)))
+                        ],
+                        clearable=False,
+                        value=3,
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div(
+                        "Force in", style={"display": "inline-block", "margin-right": "10px"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("force-in"),
+                        clearable=True,
+                        multi=True,
+                        placeholder="Select parameters to force in",
+                        value=[],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div(
+                style={"display": "grid"},
+                children=[
+                    html.Button(
+                        id=self.uuid("submit-button"),
+                        children="Update model",
```

Comment:
As we try to avoid claiming that the result is a model, should we try to avoid it all over? Think also the update button could be moved up to make it easilly visible even if e.g. the parameter and filter lists are long
```suggestion
                        children="Update",
```